expan/core/statistics.py

```diff
@@ -240,7 +240,7 @@ def pooled_std(std1, n1, std2, n2):
     :return: pooled standard deviation
     :type: float
     """
-    if not (0.5 < (std1 ** 2) / (std2 ** 2) < 2.):
+    if std1 != std2 and not (0.5 < (std1 ** 2) / (std2 ** 2) < 2.):
```

Comment:
if they're both zero, this would throw a division by zero error, wouldn't it?

Reply:
No. The whole point of this PR is to avoid exactly that

If they are both zero, the first part of the and will fail and therefore the `if` isn't used and therefore there is no warning.

The general idea here is that if the two stds are equal to each other (regardless of whether it's zero or not), then we don't need to trigger the warning that is inside the if

Reply:
Oh, sorry, I misread the 'not-equal' part. What then if `std2 == 0` is true?